docs/mkdocs.yml

```diff
@@ -202,6 +201,7 @@ nav:
           - how-tos/local-studio.md
         - LangGraph SDK: concepts/sdk.md
       - Data management:
+        - cloud/deployment/semantic_search.md
```

Comment:
This feels more appropriate here.

Reply:
hm fine with it ya! maybe should ahve a whole section on the long-term memory store

Reply:
I was actually looking for a top-level `Memory` section; I was kind of expecting one...